<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>homewindow</class>
 <widget class="QMainWindow" name="homewindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1311</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>1027</width>
      <height>611</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1016</width>
      <height>611</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>2000</width>
      <height>2000</height>
     </size>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1025</width>
       <height>609</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1311</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHome">
    <property name="title">
     <string>Home</string>
    </property>
    <addaction name="menOpHome"/>
    <addaction name="menOpSearch"/>
   </widget>
   <widget class="QMenu" name="menuAccount">
    <property name="title">
     <string>Account</string>
    </property>
    <addaction name="viewAccount"/>
   </widget>
   <addaction name="menuHome"/>
   <addaction name="menuAccount"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="menOpHome">
   <property name="text">
    <string>Home</string>
   </property>
  </action>
  <action name="menOpSearch">
   <property name="text">
    <string>Search</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="viewAccount">
   <property name="text">
    <string>View my account</string>
   </property>
  </action>
  <action name="actionEdit_my_account">
   <property name="text">
    <string>Edit my account</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
